inc/az_ulib_base.h

```diff
@@ -16,10 +16,10 @@
 #include <stdint.h>
 #else
 #include <cstdint>
-extern "C"
```

Comment:
Marcos, are you sure you want to remove this? It would make using this lib in c++ projects not as straightforward. 
It's ok if that is the plan, I just wanted to confirm.

Reply:
I'm not removing, I'm replacing it by _az_cfg_prefix.h and _az_cfg_suffix.h.

---
In reply to: [590597444](https://github.com/Azure/azure-ulib-c/pull/61#discussion_r590597444) [](ancestors = 590597444)